x/rollapp/keeper/hooks.go

```diff
@@ -0,0 +1,40 @@
+package keeper
+
+import (
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	epochstypes "github.com/osmosis-labs/osmosis/v15/x/epochs/types"
+)
+
+var _ epochstypes.EpochHooks = epochHooks{}
+
+const defaultDeleteStateInfoTimestampThreshold = time.Hour * 24
```

Comment:
can you document it?

Reply:
why? It's pretty self-explanatory innit?

Reply:
not really
maybe the name threshold is confusing, not sure why it's a threshold
also the 'good measure' thing is related

Reply:
I wasn't convinced it was needed to begin with. Now it's gone anyway